.github/workflows/build.yml

```diff
@@ -341,8 +341,19 @@ jobs:
     strategy:
       matrix:
         profiles: ${{ fromJSON(needs.dynamic-matrix.outputs.profiles) }}
+        network:
+          - name: testnet
+            snapshot_url: https://snapshots.vara-network.io/vara-testnet.snap
+            snapshot_file: vara_ci_testnet.snap
+            disable_spec_name_check: true
+            run_checks_all: false
+          - name: mainnet
+            snapshot_url: https://snapshots.vara-network.io/vara-mainnet.snap
+            snapshot_file: vara_ci.snap
+            disable_spec_name_check: false
+            run_checks_all: true
       fail-fast: false
-    name: try-runtime (${{ matrix.profiles.name }})
+    name: try-runtime (${{ matrix.profiles.name }} / ${{ matrix.network.name }})
```

Comment:
Please, use another naming or it looks like this in UI

<img width="353" height="85" alt="Image" src="https://github.com/user-attachments/assets/d4c31c3a-141e-40d2-844c-8b42fd4cc6ac" />